apps/api/src/mail/services/mail.service.ts

```diff
@@ -24,10 +24,12 @@ export class MailService implements IMailService {
     workspace: string,
     actionUrl: string,
     invitee: string,
-    role: WorkspaceRole
+    role: WorkspaceRole,
+    forRegisteredUser: boolean
```

Comment:
You would need to use this flag to decide on the type of message that would be sent to the user